examples/ts/nextjs/lib/bugsnag.js

```diff
@@ -0,0 +1,28 @@
+const Bugsnag = require('@bugsnag/js')
+const BugsnagPluginReact = require('@bugsnag/plugin-react')
+const BugsnagPluginExpress = require('@bugsnag/plugin-express')
+
+// TODO Remove this for final example
+const sharedConfig = {
+  endpoints: {
+    notify: 'http://localhost:8000',
+    sessions: 'http://localhost:10000'
+  }
+}
+
+// TODO work out how to do appVersion
+
+if (typeof window === "undefined") {
```

Comment:
As discussed, separate projects for each execution environment (node/browser). `@bugsnag/plugin-express` would get pulled into the browser bundle. We could potentially resolve that by splitting into separate files with a conditional `require` or dynamic `import()`

It gets more complicated because for this example we have a custom express server, which also uses this file (hence JS not TS).